decidim-core/lib/decidim/assets/tailwind/instance.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "ostruct"
+require "erb"
+
+module Decidim
+  module Assets
+    module Tailwind
+      class Instance
+        def write_runtime_configuration
+          evaluated_template = ERB.new(tailwind_configuration_template).result(tailwind_variables.instance_eval { binding })
+
+          File.write(File.join(app_path, "tailwind.config.js"), evaluated_template)
+        end
+
+        private
+
+        def tailwind_variables
+          # The directories where Tailwind should search for templates are the installed
+          # decidim gems and the current directory (which is the Rails app)
+          decidim_gems = Bundler.load.specs.select { |spec| spec.name =~ /^decidim-/ }
```

Comment:
The kind of array this generates now looks like this after the file is generated:
```js
  content: ['#<Gem::Specification name=decidim-api version=0.28.0.dev>','#<Gem::Specification name=decidim-core version=0.28.0.dev>','#<Gem::Specification name=decidim-comments version=0.28.0.dev>','#<Gem::Specification name=decidim-accountability version=0.28.0.dev>','#<Gem::Specification name=decidim-admin version=0.28.0.dev>','#<Gem::Specification name=decidim-assemblies version=0.28.0.dev>','#<Gem::Specification name=decidim-blogs version=0.28.0.dev>','#<Gem::Specification name=decidim-budgets version=0.28.0.dev>','#<Gem::Specification name=decidim-debates version=0.28.0.dev>','#<Gem::Specification name=decidim-forms version=0.28.0.dev>','#<Gem::Specification name=decidim-generators version=0.28.0.dev>','#<Gem::Specification name=decidim-meetings version=0.28.0.dev>','#<Gem::Specification name=decidim-pages version=0.28.0.dev>','#<Gem::Specification name=decidim-participatory_processes version=0.28.0.dev>','#<Gem::Specification name=decidim-proposals version=0.28.0.dev>','#<Gem::Specification name=decidim-sortitions version=0.28.0.dev>','#<Gem::Specification name=decidim-templates version=0.28.0.dev>','#<Gem::Specification name=decidim-surveys version=0.28.0.dev>','#<Gem::Specification name=decidim-system version=0.28.0.dev>','#<Gem::Specification name=decidim-verifications version=0.28.0.dev>','#<Bundler::StubSpecification name=decidim-bulletin_board version=0.23.0 platform=ruby>','#<Gem::Specification name=decidim-conferences version=0.28.0.dev>','#<Gem::Specification name=decidim-consultations version=0.28.0.dev>','#<Gem::Specification name=decidim-dev version=0.28.0.dev>','#<Gem::Specification name=decidim-elections version=0.28.0.dev>','#<Gem::Specification name=decidim-initiatives version=0.28.0.dev>','.']
```

This causes a parse error when trying to load the home page with redesign enabled.

So instead of printing out the gem I think you should print out the root directories for each gem, i.e. `spec.full_gem_path`. The same way that we do at the webpacker configuration:
https://github.com/decidim/decidim/blob/ea43e9c29ba6097be8d87e9e307466cc429f8ce6/decidim-core/lib/decidim/webpacker/configuration.rb#L77